### ISSUE TITLE
Armor CSV Export added a perk and left out a perk on a couple of Armor pieces.

### ISSUE THREAD

**Issue by Niborum** [DestinyItemManager/DIM#3869]
Armor Piece Warlock Bond  Hash 2088829612 has two perks on it  Hand Cannon Reserves and Heavy Ammo finder.  I exported the Armor data a second time and the CSV still showed ONLY the Hand Cannon Reserves perk.  On another piece the CSV file actually had an extra Perk versus in game. Unfortunately I deleted these piece before getting the Hash information. (Sorry)

**Comment by duckmanBR** [comment]
Trying to reproduce this, I couldn't find an example where the CSV showed less, but I did find one we showed more.
I'm on mobile comparing to a recent export, but I'll debug later.

**Comment by Niborum** [comment]
Thanks, for your help.  I will watch and if it comes up again I’ll screen shot this.   

 

Just FYI, I am importing the CSV into Access and have developed some screens and reports to help with Armor evaluation and analysis…(decide what to keep and destroy).  I am hoping to optimize my storage…(since I am a pack rat).

**Comment by Niborum** [comment]
So I found the Armor Piece that had more Perks in the CSV, BUT it turns out the issue is with Destiny 2… I was wearing an Armor piece that showed in game Machine Gun Reserves and Shotgun Reserves on it, I had Machine gun reserves active.  When I decided to change to the Shotgun Reserves on my character the in game machine gun reserves disappeared.   Basically DIM reported exactly what Destiny 2 was showing although it was a faux perk.  Which leads me to wonder if I did actually have machine gun reserves at that time or not….  Hmmm

 

Thanks gain for all your efforts and support….and also for supporting the St. Jude Hospitals

 

Take care
